src/FormMigration/Steps/FormTaxonomies.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+
+/**
+ * @unreleased
+ */
+class FormTaxonomies extends FormMigrationStep
```

Comment:
Should we add a `canHandle` method?  Potentially 

```taxonomy_exists('give_forms_tag') || taxonomy_exists('give_forms_category')```

Reply:
The `canHandle` method would make sense if we split forms, categories unto separate tests.
But, this being in a single test this is a "and/or" situation.